tests/test_tracking_pin_cell/test_tracking_pin_cell.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import math
+from collections import OrderedDict
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, 'openmoc'))
+from testing_harness import TrackingTestHarness
+from input_set import PinCellInput
+
+import openmoc
+
+class TrackingPinCellTestHarness(TrackingTestHarness):
+    """Tests tracking over a pin cell geometry."""
+
+    def __init__(self):
+        super(TrackingPinCellTestHarness, self).__init__()
+        self.input_set = PinCellInput()
+
+    def _setup(self):
+        """Initialize the materials, geometry, and tracks."""
+        super(TrackingPinCellTestHarness, self)._create_geometry()
+
+        # Initialize track objects
+        tracks = OrderedDict()
```

Comment:
I would put this initialization in the parent class' `__init__()` so that `OrderedDict` only has to be imported into one file. Then each child class will simply populate `self.tracks` rather than `tracks`.